modules/model-metadata-bias-monitoring.adoc

```diff
@@ -0,0 +1,26 @@
+:_module-type: PROCEDURE
+
+[id="model-metadata-bias-monitoring_{context}"]
+= Examining TrustyAI's model metadata
+
+To verify that TrustyAI sees the models via the `/info` endpoint:
+
+1. Find the route to the TrustyAI Service:
++
+[source]
+----
+ TRUSTY_ROUTE=https://$(oc get route/trustyai-service --template={{.spec.host}}); echo $TRUSTY_ROUTE
+----
+
+2. Query the `/info` endpoint:
++
+[source]
+----
+curl -H "Authorization: Bearer ${TOKEN}" $TRUSTY_ROUTE/info | jq
+----
+This will output a json file (link:https://github.com/trustyai-explainability/odh-trustyai-demos/blob/main/2-BiasMonitoring/resources/info_response.json[sample provided here]) 
```

Comment:
remove this link and replace it with a pointer to the local version that they downloaded: 2-BiasMonitoring/resources/info_response.json